Code/Engine/Core/World/World.h

```diff
@@ -251,9 +251,12 @@ class EZ_CORE_DLL ezWorld final
   ///@}
 
   /// \brief If enabled, the full simulation should be executed, otherwise only the rendering related updates should be done
+  /// Note the state returned by \link GetWorldSimulationEnabled() will not reflect state of \param bEnable immediately,
```

Comment:
There is no need to \link we don't use it elsewhere, links will be automatically inserted.

Reply:
Fixed in 63e08d6